lib/diyepw/diyepw/meteorology.py

```diff
@@ -0,0 +1,263 @@
+import numpy as np
+import pandas as pd
+import os
+import datetime
+import calendar
+import typing
+
+_RANGES = {
```

Comment:
make `RANGES` be a class variable inside `Meterology` not outside

Reply:
Done